package.json

```diff
@@ -12,6 +12,9 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
+        "npm": "2.15.10"
```

Comment:
Have you tested this out with `npm@3`. I didn't have much issues (if any) from moving to `2->3` when 3 started rolling out but I don't know if it has some troubles with how Brackets resolves the extension dependencies.